fingertree-rm/fingertree-rm.cabal

```diff
@@ -0,0 +1,93 @@
+cabal-version:       3.0
+name:                fingertree-rm
+version:             0.1.0.0
```

Comment:
Do we want to use the same versioning policy as in consensus?

Reply:
I think that makes sense. I consider `0.1.0.0` a placeholder for now, and I'll add a document explaining/referring to the versioning policy in a later PR.